### PR TITLE
update batch send example

### PR DESCRIPTION
**Pull request by bryanklewis**
### Fix or Enhancement?
Fix, the sendBatch example is out of date. Added clarifying statement that adds some intended use. This would resolve issue #127 

readme/documentation change only.

- [x] All tests passed
- [x] Add change to `changelog.md`

### Environment
- OS: Windows
- Go version: 1.12.6

### CODE REVIEW

**Comment by bryanklewis** [comment]
I don't think the changelog really needs updated for something like this, up to you Azure folks on that one.

File: readme.md [hunk 1]
```diff
@@ -228,15 +228,14 @@ particular partition. You can do this in two ways.
...
 ```go
-batch := &EventBatch{
-            Events: []*eventhub.Event { 
-                eventhub.NewEventFromString("one"),
-                eventhub.NewEventFromString("two"),
-            },
-        }
-err := client.SendBatch(ctx, batch)
+var events []*Event
+events = append(events, NewEventFromString("one"))
```
Comment (gavinfish) [inline readme.md]: Add package name for public method?  `eventhub.NewEventFromString("one")`, `eventhub.NewEventBatchIterator(events...)`

Comment (bryanklewis) [reply readme.md]: I was keeping it generic, but it does add clarity. If we add the package name i think we should show the import and do it for all objects that would need it.

Comment (gavinfish) [reply readme.md]: I got your point. But I just want to make sure all the samples in the readme have similar format.

Comment (bryanklewis) [reply readme.md]: added the package name

**Pull request merged**